docs/node-operators/get-started/setup-validator.md

```diff
@@ -302,37 +302,27 @@ Once you have tokens, connect your Ethereum wallet and your Vega Wallet, and ass
 The association will need to be confirmed by both the Ethereum and Vega blockchains, and may take some time.
 
 ## Announce node on-chain
-Use your Ethereum key to announce your node to the network.
-
-You'll need to know the [current epoch ↗](https://validator-testnet.governance.vega.xyz/staking), and have the following data to hand: the URL for your validator website, and URL for the avatar that will show up on the governance dApp next to your node name.
+You'll need to know the [current epoch ↗](https://validator-testnet.governance.vega.xyz/staking), and have the following data to hand: the URL for your validator website, and the URL for the avatar that will show up on the governance dApp next to your node name.
 
 ```shell
-vega announce_node --home="YOUR_VEGA_HOME_PATH" --info-url="YOUR_VALIDATOR_URL" --avatar-url="YOUR_AVATAR_URL" --country="UK" --name="YOUR_NODE_NAME" --from-epoch="CURRENT_EPOCH" --submitter-address="YOUR_ETHEREUM_KEY"
+vega announce_node --home="YOUR_VEGA_HOME_PATH" --info-url="YOUR_VALIDATOR_URL" --avatar-url="YOUR_AVATAR_URL" --country="UK" --name="YOUR_NODE_NAME" --from-epoch="CURRENT_EPOCH"
 ```
 
+The optional argument `--submitter-address` can be used to set a Ethereum key that can be used to help maintain the multisig-control contract.
```

Comment:
**question (non-blocking)**: Why would you choose to set this? Why would you choose not to?

Reply:
Fleshed it out a bit, with a little mention about rewards to help get the readers attention.